build.sbt

```diff
@@ -865,3 +879,17 @@ lazy val shadingSettings = Def.settings(
   ).map(ShadingRule.moveUnder(_, "scala.meta.shaded.internal")),
   validNamespaces ++= Set("scala.meta", "org.scalameta", "scala", "java")
 )
+
+def platformPublishSettings(platform: Platforms.Platform) =
+  if (Platforms.buildPlatform(platform))
+    Def.settings()
+  else
+    Def.settings(
```

Comment:
I don't know how much of it is needed, just copied, with some mods, from https://gist.github.com/joescii/4431cb2185eddb699dfb79b9c910e333